upstream-dev pint tests are failing

```
________________ ERROR collecting cf_xarray/tests/test_units.py ________________
cf_xarray/tests/test_units.py:11: in <module>
    from ..units import units as ureg
cf_xarray/units.py:81: in <module>
    pint.unit.UnitDefinition("percent", "%", (), pint.converters.ScaleConverter(0.01))
E   AttributeError: module 'pint' has no attribute 'unit'
________________ ERROR collecting cf_xarray/tests/test_units.py ________________
cf_xarray/tests/test_units.py:11: in <module>
    from ..units import units as ureg
cf_xarray/units.py:81: in <module>
    pint.unit.UnitDefinition("percent", "%", (), pint.converters.ScaleConverter(0.01))
E   AttributeError: module 'pint' has no attribute 'unit'
=================================== FAILURES ===================================
_________________________ test_coordinates_quantified __________________________
[gw1] linux -- Python 3.10.4 /usr/share/miniconda/envs/cf_xarray_test/bin/python
    @requires_pint
    def test_coordinates_quantified():
        # note: import order is important
>       from .. import units  # noqa
cf_xarray/tests/test_accessor.py:222: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
cf_xarray/units.py:17: in <module>
    def short_formatter(unit, registry, **options):
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
func = <function short_formatter at 0x7f990adb84c0>
    def wrapper(func):
        if name in _FORMATTERS:
>           raise ValueError(f"format {name:!r} already exists")  # or warn instead
E           ValueError: Invalid format specifier
/usr/share/miniconda/envs/cf_xarray_test/lib/python3.10/site-packages/pint/formatting.py:160: ValueError
```

--- a/cf_xarray/units.py
+++ b/cf_xarray/units.py
@@ -10,12 +10,6 @@ from pint import (  # noqa: F401
     UnitStrippedWarning,
 )
 
-try:
-    from pint.facets.plain import UnitDefinition, ScaleConverter
-except ImportError:
-    from pint.unit import UnitDefinition
-    from pint.converters import ScaleConverter
-
 # from `xclim`'s unit support module with permission of the maintainers
 try:
 
@@ -83,9 +77,7 @@ units = pint.UnitRegistry(
     force_ndarray_like=True,
 )
 
-units.define(
-    UnitDefinition("percent", "%", (), ScaleConverter(0.01))
-)
+units.define("percent = 0.01 = %")
 
 # Define commonly encountered units (both CF and non-CF) not defined by pint
 units.define("@alias meter = gpm")

